File: TransportAgencyWebAPI/Models/ViewModels/FindTripInfoViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using TransportAgencyWebAPI.Models;

namespace TransportAgencyWebAPI.Models.ViewModels
{
	public class FindTripInfoViewModel
	{
		public Guid ArrivalId { get; set; }
		public DateTime? ArrivalDate { get; set; }
		public Guid DepartureId { get; set; }
		public DateTime DepartureDate { get; set; }
		public Guid? TranportTypeId { get; set; }
	}
}

File: TransportAgencyAngular/src/app/viewModel/findTripInfo.viewModel.ts
import {Guid} from 'guid-typescript';
import {InjectionToken} from '@angular/core';

export const FIND_INFO = new InjectionToken('find_info');
export class FindTripInfoViewModel {
  arrivalId: Guid;
  arrivalDate?: Date;
  departureId: Guid;
  departureDate: Date;
  transportTimeId: Guid;
}
